lib/base/networking/api/handler/auth_handler.dart

```diff
@@ -106,12 +118,11 @@ class AuthHandler {
       if (url != null && url.toString().startsWith(Routes.ssoRedirect)) {
         _logger.i('Web view loaded URL: $url');
         await _handleCookieRetrieval(url);
-        isLoginSuccessful = true; // Set flag to true on successful login
+        isLoginSuccessful = false; // Set flag to true on successful login
         // Due to the token being signed from TUM Live RBG, the app will not be able to decode it
```

Comment:
Shouldn't `isLoginSuccessful` be set to `true`?

Reply:
✅